app/services/search/indices.rb

```diff
@@ -0,0 +1,19 @@
+# This class is used for Postgres FTS spike only
+module Search
+  class Indices
+    MULTISEARCH_CLASSES = [Article, Comment, PodcastEpisode].freeze
+
+    def self.clear!(klass)
+      PgSearch::Document.delete_by(searchable_type: klass.name)
+    end
+
+    def self.rebuild!(klass)
+      PgSearch::Multisearch.rebuild(klass)
+    end
+
+    def self.rebuild_all!
+      MULTISEARCH_CLASSES.each { |klass| PgSearch::Multisearch.rebuild(klass) }
+      nil
+    end
```

Comment:
Got tired of typing `.rebuild(Article|PodcastEpisode|Comment)` each time I had to rebuild all of them. This would eventually be split in an inline version and an async one I guess